services/minespace-web/src/components/pages/Project/ProjectPage.js

```diff
@@ -70,7 +70,7 @@ export class ProjectPage extends Component {
     }
   }
 
-  handleTabChange = (activeTab, irtStatus) => {
+  handleTabChange = (activeTab, irtStatus, mmaStatus) => {
```

Comment:
Is "mmaStatus" being used in this function?

Reply:
Left from previous change. Removing